src/code42cli/cmds/devices.py

```diff
@@ -0,0 +1,368 @@
+from collections import OrderedDict
+from datetime import date
+from datetime import datetime
+
+import click
+from pandas import DataFrame
+from pandas import to_datetime
+from pandas import to_timedelta
+from py42 import exceptions
+
+from code42cli.bulk import run_bulk_process
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.errors import Code42CLIError
+from code42cli.file_readers import read_csv_arg
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.output_formats import OutputFormatter
+
+
+@click.group(cls=OrderedGroup)
+@sdk_options(hidden=True)
+def devices(state):
+    """For managing computers within Code42."""
```

Comment:
Is `within Code42` more apt or `within an organisation`? Similarly in all places elsewhere.

Reply:
The commands don't require an organization UID and will default to showing all devices within your Code42 environment (i.e. all that are visible to your user). For that reason I didn't specify "organization" within the docstrings.

I could update it to say "organization", or if we want to be exactingly correct we could say "within a Code42 tenant" or something like that.

Reply:
I'd still say `For managing devices within Code42` because no other part of our docs in py42 or the CLI refer to them as `computers`.

Reply:
I've updated this text to use 'devices' and to say "within your Code42 environment", which is more specific than "within Code42" while still accurately reflecting that it is not limited to a single organization